src/app/blog/index.html

```diff
@@ -4,6 +4,37 @@
   windowTitle="SKY UX Blog">
 
 	<article class="blog-post">
+		<header>
+			<h2 class="sky-section-heading">Design update — May 2019</h2>
+		</header>
+		<section>
+			<h3 class="sky-subsection-heading">Recently completed</h3>
+				<h4 class="sky-emphasized">Bugs</h4>
+					<p>We spent a lot of time this month cleaning up bugs.</p>
+        <h4 class="sky-emphasized">Design proposals</h4>
+					<ul>
+						<li><a href="https://github.com/blackbaud/skyux-forms/issues/40">Input field character counts</a></li>
+					</ul>
+		</section>
+		<section>
+			<h3 class="sky-subsection-heading">In the works</h3>
+				<h4 class="sky-emphasized"><a href="https://github.com/blackbaud/skyux-phone-field/issues/3">Phone input field</a></h4>
+					<p>We are porting the phone input field from SKY UX 1. We currently have a release candidate, but we are changing the country selector UI before the official release.</p>
+        <h4 class="sky-emphasized"><a href="https://github.com/blackbaud/skyux-list-builder/issues/22">List view switcher</a></h4>
+					<p>We are porting the list view switcher from SKY UX 1.</p>
+        <h4 class="sky-emphasized"><a href="https://github.com/blackbaud/skyux2/issues/2331">Data entry grid</a></h4>
+          <p>We are working on SKY UX-themed stylesheets and configuration files for use with the ag-grid library.</p>
+        <h4 class="sky-emphasized"><a href="https://github.com/blackbaud/skyux2/issues/724">Split view</a></h4>
+          <p>We are building a responsive container to hold a list of items and a workspace in which the user can complete tasks for the selected item.</p>
```

Comment:
No big deal if you disagree, but I'd change "a workspace in which the user can complete" to "a workspace for users to complete."